application_form/validators.py

```diff
@@ -48,3 +52,27 @@ def _calculate_control_character(self, individual_number: str) -> str:
         number = int(self.date_of_birth.strftime("%d%m%y") + individual_number)
         index = number % len(alphabet)
         return alphabet[index]
+
+
+class ProjectApplicantValidator:
+    def __call__(
+        self,
+        project_uuid: UUID,
+        date_of_birth_and_ssn_suffix: Union[Tuple[date, str], List[Tuple[date, str]]],
+    ):
+        if isinstance(date_of_birth_and_ssn_suffix, Tuple):
+            date_of_birth_and_ssn_suffix = [date_of_birth_and_ssn_suffix]
+        if isinstance(date_of_birth_and_ssn_suffix, List):
+            date_of_birth_and_ssn_suffix = date_of_birth_and_ssn_suffix.copy()
+
+        item = date_of_birth_and_ssn_suffix.pop()
+        queryset = Applicant.objects.filter(date_of_birth=item[0], ssn_suffix=item[1])
```

Comment:
As you already know, or at least assume here that the first value is a date and the second one a string. It would be nice for readability if you split these up into variables before using the `item` here. That way you could do `date_of_birth, ssn_suffix = item` instead of using the indices for the `item`.